packages/lib-classifier/src/components/Classifier/ClassifierContainer.js

```diff
@@ -209,7 +207,7 @@ export default function ClassifierContainer({
             <Classifier
               locale={locale}
               onError={onError}
-              project={project}
+              project={project} // This isn't used in Classifier, remove?
```

Comment:
I noticed that the Classifier component does not use a `project` prop. Is it somehow used elsewhere or can we remove it?

Reply:
Removed in #5038.